### PR TITLE
add option to enable profiling

### PR DESCRIPTION
**Pull request by pheanex**
add option to enable profiling

### CODE REVIEW

File: kube_log_watcher/agents/scalyr.py [hunk 1]
```diff
@@ -91,6 +91,7 @@ def __init__(self, cluster_id: str, load_template):
         self.dest_path = os.environ.get('WATCHER_SCALYR_DEST_PATH')
         self.scalyr_server = os.environ.get('WATCHER_SCALYR_SERVER')
         self.parse_lines_json = os.environ.get('WATCHER_SCALYR_PARSE_LINES_JSON', False)
+        self.enable_profiling = os.environ.get('WATCHER_SCALYR_ENABLE_PROFILING', False)
```
[the remark this answers is not in the document]
Comment (pheanex) [reply kube_log_watcher/agents/scalyr.py]: Good thinking, fixed it for both

**Comment by pheanex** [comment]
:+1:

**Comment by astarnell** [comment]
:+1:

**Comment by pheanex** [comment]
:+1:

**Comment by anton-ryzhov** [comment]
:+1:

**Pull request merged**